gcloud/monitoring/metric.py

```diff
@@ -28,26 +25,32 @@
 
 
 class MetricKind(object):
-    """Allowed values for the `kind of measurement`_.
+    """Choices for the `kind of measurement`_.
 
     .. _kind of measurement:
         https://cloud.google.com/monitoring/api/ref_v3/rest/v3/\
         projects.metricDescriptors#MetricKind
     """
 
+    METRIC_KIND_UNSPECIFIED = 'METRIC_KIND_UNSPECIFIED'
+    """Not allowed in metric descriptors."""
```

Comment:
Thanks. Can you give a screenshot of how that renders with Sphinx? (I can clone your branch if need be.)